performance_test/helper_scripts/performance_test_file_reader_pf.py

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env python
```

Comment:
This file must not be added as all relevant information are already plotted by the revamped polling script. But please double check.

Reply:
This one: https://github.com/ApexAI/performance_test/tree/master/performance_test/helper_scripts/apex_performance_plotter

I just realized that a link from the main README is missing and the old plotter script was not removed.

@jpsamper2009: Was there a reason that the old plotter script was not removed after the new file plotter was added?

Reply:
I removed it from this MR